proc-macros/tests/ui/correct/only_client.rs

```diff
@@ -1,14 +1,14 @@
 //! Example of using proc macro to generate working client.
 
-use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use jsonrpsee::{core::Error, proc_macros::rpc};
```

Comment:
```suggestion
use jsonrpsee::proc_macros::rpc;
```

Reply:
Git seems to complain about this, although we do end up using it in the macro

Reply:
it's probably because we overwrite the error type in the client and rustc understands that :P